app/models/project.rb

```diff
@@ -1,4 +1,5 @@
 class Project < ApplicationRecord
   validates :name, :description, presence: true
```

Comment:
You can write it on one line:
```suggestion
  validates :name, :description, presence: true, uniqueness: true
```